### PR TITLE
Update get_started.md: Point to the new PPAs repositories

### PR DESCRIPTION
**Pull request by jlblancoc**
Refer to the new PPA repository instead

### CODE REVIEW

**Comment by jlblancoc** [comment]
cc: @dellaert , @berndpfrommer, I think we are ready for this now, right?

**Comment by jlblancoc** [comment]
PS: I've been using the nightly builds PPA for a while without issues.

File: get_started.md [hunk 1]
```diff
@@ -23,20 +23,27 @@ Optional prerequisites - used automatically if findable by CMake:
...
+sudo apt install libgtsam-dev libgtsam-unstable-dev
 ```
 
-#### Install GTSAM (unstable)
+#### Add PPA for the latest GTSAM 4.x stable release
 
 ```sh
-sudo apt install libgtsam-dev
+# Add PPA
+sudo add-apt-repository ppa:joseluisblancoc/gtsam-release-4.0
```
Comment (varunagrawal) [inline get_started.md:43]: Is there not a PPA on the borglab account? We know and trust you but a random user might find this suspicious.

Comment (berndpfrommer) [reply get_started.md:43]: Yes, there is such a repo:

ppa:borglab/gtsam-release-4.0

**Comment by berndpfrommer** [comment]
I think the automated packaging works well enough now to go live with it. In case the nightly builds fail, the snapshot will become somewhat stale, but whatever is in the repo should still work.

File: get_started.md (hunk 1 above)
Comment (jlblancoc) [reply get_started.md]: Ooops... copy/paste typo, of course. It's fixed now.

**Review by varunagrawal — APPROVED** [review]
LGTM

**Pull request merged**